.github/workflows/build.yml

```diff
@@ -3,8 +3,18 @@ name: Build
 on:
   push:
     branches: [ main ]
+    paths:
+      - '**.go'
+      - 'go.mod'
+      - 'go.sum'
+      - .github/workflows/build.yml
   pull_request:
     branches: [ main ]
+    paths:
+      - '**.go'
+      - 'go.mod'
+      - 'go.sum'
+      - .github/workflows/build.yml
```

Comment:
Rust/Cargo/BPF would need to trigger the build workflow as well

Reply:
I'll handle it in #510